src/types/index.ts

```diff
@@ -34,30 +37,38 @@ export interface FsNode {
   type: FileType;
 }
 
-export interface WatchFileWatchers {
-  onChange: (newContent: string) => void;
+export interface MoveEvent {
+  eventType: 'MOVE';
+  node: FsNode;
+  to: string;
```

Comment:
we need to add this on repl-it-web's side

Reply:
wdym?

Reply:
`to` is not sent on the other side